contracts/0.4.24/Lido.sol

```diff
@@ -206,29 +130,25 @@ contract Lido is Versioned, StETHPermit, AragonApp {
     // Staking limit was removed
     event StakingLimitRemoved();
 
-    // Emits when validators number delivered by the oracle
-    event CLValidatorsUpdated(
-        uint256 indexed reportTimestamp,
-        uint256 preCLValidators,
-        uint256 postCLValidators
-    );
+    // Emitted when validators number delivered by the oracle
+    event CLValidatorsUpdated(uint256 indexed reportTimestamp, uint256 preCLValidators, uint256 postCLValidators);
 
-    // Emits when var at `DEPOSITED_VALIDATORS_POSITION` changed
-    event DepositedValidatorsChanged(
-        uint256 depositedValidators
-    );
+    // Emitted when var at `DEPOSITED_VALIDATORS_POSITION` changed
+    event DepositedValidatorsChanged(uint256 depositedValidators);
 
-    // Emits when oracle accounting report processed
+    // Emitted when oracle accounting report processed
+    // @dev `principalCLBalance` is the balance of the validators on previous report
+    // plus the amount of ether that was deposited to the deposit contract since then
     event ETHDistributed(
         uint256 indexed reportTimestamp,
-        uint256 preCLBalance,
+        uint256 principalCLBalance, // preClBalance + deposits
```

Comment:
Return back the `preClBalance` because of the back compatibility.

Reply:
Also, need update https://github.com/lidofinance/lido-oracle/pull/581/files#diff-40409c1972c071a18cba4721985771741da3d6baebb08e3d0397e7699ddaced0

Reply:
Fixed in 28139c8cece6fa4d37fa78b625e174d4cbb85c5c